CHANGELOG.md

```diff
@@ -40,6 +84,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- Fixed IAM permissions issue with `<prefix>-postgres-migration-async-operation` Lambda
```

Comment:
This is a duplicate and should be removed

Reply:
Updated as commented